src/Book.java

```diff
@@ -1,32 +1,32 @@
 import java.io.Serializable;
 
+//File ready for static review
+
 public class Book implements Serializable {
 
     private String title;
     private String author;
-    private String callNo;
+    private String callNumber;
     private int bookId;
     private enum State {AVAILABLE, ON_LOAN, DAMAGED, RESERVED}
-    private State state;
+    private State bookState;
 
 
     public Book(String author, String title, String callNo, int id) {
         this.author = author;
         this.title = title;
-        this.callNo = callNo;
+        this.callNumber = callNo;
         this.bookId = id;
-        this.state = State.AVAILABLE;
+        this.bookState = State.AVAILABLE;
     }
 
 
     public String toString() {
-        StringBuilder stringBuilder = new StringBuilder();
-        stringBuilder.append("book: ").append(bookId).append("\n")
-                .append("  Title:  ").append(title).append("\n")
-                .append("  Author: ").append(author).append("\n")
-                .append("  CallNo: ").append(callNo).append("\n")
-                .append("  State:  ").append(state);
-        return stringBuilder.toString();
+        return "book: " + bookId + "\n" +
+                "  Title:  " + title + "\n" +
```

Comment:
Line 26 to 29 are not indented with an increment of 4 spaces. They are currently double-indented

Reply:
"When there are multiple continuation lines, indentation may be varied beyond +4 as desired." Google Java Style Guide 4.5.2

Reply:
Confirmed and accepted.